src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+public class Task {
+    private String taskName;
+    private boolean isDone;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+    public void setDone() {
+        isDone = true;
+    }
+    public void setUndone() {
+        isDone = false;
+    }
+
+    public String toString() {
```

Comment:
Perhaps this expression can be stored in a variable? This will help you avoid complicated expressions